script/create-dist.py

```diff
@@ -133,7 +133,8 @@ def copy_license():
 
 def create_api_json_schema():
   outfile = os.path.join(DIST_DIR, 'electron-api.json')
-  execute(['electron-docs-linter', '--outfile={0}'.format(outfile)])
+  execute(['electron-docs-linter', '--outfile={0}'.format(outfile)],
+           '--version={}'.format(ELECTRON_VERSION.replace('v', '')))
```

Comment:
does this new string need to go into the args array? Looks like the `]` is still after the `outfile` argument.